adapters/medianet/medianet.go

```diff
@@ -89,8 +89,13 @@ func Builder(bidderName openrtb_ext.BidderName, config config.Adapter, server co
 	}, nil
 }
 
-func getMediaTypeForImp(impID string, imps []openrtb2.Imp) (openrtb_ext.BidType, error) {
-	mediaType := openrtb_ext.BidTypeBanner
+func getMediaTypeForImp(bid *openrtb2.Bid, imps []openrtb2.Imp) (openrtb_ext.BidType, error) {
+	mediaType, err := getBidMediaTypeFromMtype(bid)
+	if err == nil {
+		return mediaType, nil
```

Comment:
Should not ignore error returned from `getBidMediaTypeFromMtype` function. This should be - 
`if err != nil {
     return nil, err
}`

Reply:
Removed the fallback. Pls check